<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reveil</class>
 <widget class="QDialog" name="Reveil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>302</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Heure</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit"/>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="text">
       <string>Répéter :</string>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>545</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="lundi_CheckBox">
      <property name="text">
       <string>Lundi</string>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mardi_CheckBox">
      <property name="text">
       <string>Mardi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="mercredi_CheckBox">
      <property name="text">
       <string>Mercredi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="jeudi_CheckBox">
      <property name="text">
       <string>Jeudi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="vendredi_CheckBox">
      <property name="text">
       <string>Vendredi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="samedi_CheckBox">
      <property name="text">
       <string>Samedi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="dimanche_CheckBox">
      <property name="text">
       <string>Dimanche</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
